docs/pages/build-reference/how-tos.md

```diff
@@ -80,6 +80,9 @@ e.g.
 
 ## How to use git submodules
 
+If you are using default VCS workflow, the content of your working directory will be uploaded to EAS as it is, including the content of your submodules. If you are building on CI you will need to initialize them, otherwise empty directories will be uploaded.
```

Comment:
```suggestion
If you are using the default VCS workflow, the content of your working directory will be uploaded to EAS Build as it is, including the content of Git submodules. If you are building on CI you will need to initialize them, otherwise, empty directories will be uploaded.
```